### DISCUSSION TITLE
Does PyGWalker implement a "Total" field?

### DISCUSSION THREAD

**Discussion by manoelpqueiroz** [Kanaries/pygwalker#559] [Q&A] [answered]
Very straightforward question: especially in table views, it would be nice to be able to see the total amounts when breaking down a measure by a certain dimension:

![image](https://github.com/Kanaries/pygwalker/assets/23669137/ef4511f0-9fc3-4df4-8a68-7b8c8fe97c6a)

Does PyGWalker implement this option? I haven't found anything in the documentation or looking for the term "total" on the Issues section and here in the Discussions.

**Comment by ObservedObserver** [comment, marked as answer]
Yes, PyGWalker had already implement this feature, but due to internal bug, the feature now is not displayed to end users. We have just finished fixing it here https://github.com/Kanaries/graphic-walker/pull/388. And the "total" feature will be avaiable in next release (0.4.8.7) within 24 hours. So you can use it pretty soon.

**Reply by manoelpqueiroz** [reply]
Awesome! Thanks a lot for the info! And congratulations on the project, it's really amazing to see open source alternatives to commercial software delivering on functionality. Keep up the good work!

**Reply by longxiaofei** [reply]
The fix is released in 0.4.8.8. `pip install pygwalker==0.4.8.8`
